test_conformance/select/test_select.c

```diff
@@ -582,6 +587,11 @@ int main(int argc, char* argv[])
         log_error( "Failed to allocate memory for argList array.\n" );
         return 1;
     }
+    argc = parseCustomParam(argc, argv);
```

Comment:
This should be a separate change (see https://github.com/KhronosGroup/OpenCL-CTS/issues/327). There is a patch on the gitlab `cl21_Arm` branch that we haven't upstream yet IIRC.

Reply:
sure